### PR TITLE
Fix export 

### PR DESCRIPTION
**Pull request by jamesRUS52**
Export with hidden and visibleInExport = true not working.
It was used default hidden value from definition, not toggled state. So, it not will be able to be exported.
Fixed and changed logic. Three state of Visible in export var.
null - export depend on toggled state.
true - always in export
false - always absent in export

### CODE REVIEW

File: src/Traits/Exportable.php [hunk 1]
```diff
@@ -100,9 +100,17 @@ private function export(string $exportableClass, bool $selected)
          */
         $exportable = new $exportableClass();
 
+        $current_hidden_states = collect($this->columns)
```
Comment (luanfreitasdev) [inline src/Traits/Exportable.php:103]: Let's follow the camel case pattern

**Comment by luanfreitasdev** [comment]
Thanks for submitting your PR!

**Review by luanfreitasdev — CHANGES_REQUESTED** [review]
Please fix the style of the variable

File: src/Traits/Exportable.php (hunk 1 above)
Comment (jamesRUS52) [reply src/Traits/Exportable.php]: done

**Review by luanfreitasdev — CHANGES_REQUESTED** [review]
Method transform

File: src/Traits/Exportable.php [hunk 2]
```diff
@@ -100,10 +100,10 @@ private function export(string $exportableClass, bool $selected)
          */
         $exportable = new $exportableClass();
 
-        $current_hidden_states = collect($this->columns)
+        $currentHiddenStates = collect($this->columns)
             ->mapWithKeys(fn ($column) => [$column['field'] => $column['hidden']]);
-        $columns_with_hidden_state = array_map(function ($column) use ($current_hidden_states) {
-            $column->hidden = $current_hidden_states[$column->field];
+        $columns_with_hidden_state = array_map(function ($column) use ($currentHiddenStates) {
```
Comment (luanfreitasdev) [inline src/Traits/Exportable.php:105]: here too

Comment (jamesRUS52) [reply src/Traits/Exportable.php]: done

Comment (luanfreitasdev) [reply src/Traits/Exportable.php]: $columns_with_hidden_state

**Comment by luanfreitasdev** [comment]
Great job

**Review by luanfreitasdev — APPROVED** [review]

**Pull request merged**